.github/label-conflict.yml

```diff
@@ -0,0 +1,40 @@
+name: Auto Label Conflicts
+on:
+  push:
+    branches:
+      - 'develop'
+      - '4.*'
```

Comment:
Should be removed?

Reply:
Just PR, no needed push trigger. I will change